lang/ui.fr.json

```diff
@@ -859,15 +859,15 @@
     "description": "Simulator gesture option"
   },
   "simulator-gesture-down": {
-    "defaultMessage": "logo en bas",
```

Comment:
This is expected change as we used some not-quite-right translations from Reference as a stop gap.